src/signingClient.spec.ts

```diff
@@ -97,13 +98,82 @@ describe('queryClient', () => {
       },
     );
     const hash = await computeTransactionHash(signedTxRaw as TxRaw);
-    expect(hash).toEqual('AE12BE64B836DD622747CABBB7CD4D68441AB9C3F38CC6E2F62E1DFFC23FD38D');
+    expect(hash).toEqual('79EF19B2E76F5A2066E753D1F60D30174263C4EB73CBEEE1339A9BD18F80FE6A');
     const signedTxRaw2 = await client.createISCNRecord(
       wallet.address, testData2, {
         broadcast: false, sequence: 2, accountNumber: 0, chainId: 'likecoin-mainnet-2',
       },
     );
     const hash2 = await computeTransactionHash(signedTxRaw2 as TxRaw);
-    expect(hash2).toEqual('23636A1D0EC3BF0C5B1A580227C45EC36B7049D4C7E20F152584F90A324571CF');
+    expect(hash2).toEqual('EE5515A0798E8CC26EFB62851ACFFFCC20C1423FF8088F07DAD1A07071A98C85');
+  });
+});
+
+describe('signingClient NFT', () => {
+  test('Sign create NFT class', async () => {
+    const { client, wallet } = await getSigner();
+    const signedTxRaw = await client.createNFTClass(
+      wallet.address,
+      ISCN_ID,
+      { name: 'Liker NFT #1', metadata: { a: 'b' } },
+      undefined,
+      {
+        broadcast: false, sequence: 1, accountNumber: 0, chainId: 'likecoin-mainnet-2',
+      },
+    );
+    const hash = await computeTransactionHash(signedTxRaw as TxRaw);
+    expect(hash).toEqual('02B36E2D6EF3CEC7E0D388AB3A469920CBAA292095DAC44F7ED05E0F926905BB');
+  });
+
+  // test('Sign mint NFT', async () => {
+  //   const { client, wallet } = await getSigner();
+  //   const signedTxRaw = await client.mintNFTs(
+  //     wallet.address,
+  //     NFT_CLASS_ID,
+  //     [...Array(100).keys()].map((i) => ({
+  //       id: `testing-321321-${i}`,
+  //       uri: 'testing-12341234',
+  //       metadata: {
+  //         a: 'b',
+  //       },
+  //     })),
+  //     {
+  //       broadcast: false, sequence: 1, accountNumber: 0, chainId: 'likecoin-mainnet-2',
+  //     },
+  //   );
+  //   const hash = await computeTransactionHash(signedTxRaw as TxRaw);
+  //   expect(hash).toEqual('28787C77CD79F0C292B2339DC6480622FC6713971B8E45875126F501AE3CBEFD');
+  // });
```

Comment:
Do we intend to keep this commented test?

Reply:
yes, but we can't run it with current mainnet yet since no x/nft
when we have time we should stub/make a temp chain for unit test